admin/custom.js

```diff
@@ -523,7 +523,7 @@ if (typeof customPostInits !== 'undefined') {
 
             Input.number_max.keyup(function () {
                 // Nur Nummern zulassen
-                let allowedSigns = /[^0-9]/;
+                let allowedSigns = /[^0-9\-]/;
```

Comment:
This would allow inputs like 
12-345-678-876
I would suggest starting of with a regex like this [https://regex101.com/library/pH8cJ8](https://regex101.com/library/pH8cJ8) and maybe allowing infinitely many decimal places or make some other minor adjustments.